packages/neon/neon/lib/src/widgets/image.dart

```diff
@@ -255,15 +215,76 @@ class NeonApiImage extends StatelessWidget {
     this.svgColorFilter,
     this.errorBuilder,
     super.key,
-  }) : account = null;
+  });
 
-  /// Creates a new Neon API image for a given account.
+  /// Account to use for the request.
   ///
-  /// Use [NeonApiImage] to fetch the image with the currently active account.
-  const NeonApiImage.custom({
-    required this.getImage,
-    required this.cacheKey,
-    required Account this.account,
+  /// Defaults to the currently active account in [AccountsBloc.activeAccount].
+  final Account? account;
+
+  /// Image downloader.
+  final ApiImageDownloader getImage;
+
+  /// Cache key used for [NeonCachedImage.key].
+  final String cacheKey;
+
+  /// Custom cache reviver function.
+  final CacheReviver? reviver;
+
+  /// Custom cache writer function.
+  final CacheWriter? writeCache;
+
+  /// {@macro NeonCachedImage.svgHint}
+  final bool isSvgHint;
+
+  /// {@macro NeonCachedImage.size}
+  final Size? size;
+
+  /// {@macro NeonCachedImage.fit}
+  final BoxFit? fit;
+
+  /// {@macro NeonCachedImage.svgColorFilter}
+  final ColorFilter? svgColorFilter;
+
+  /// {@macro NeonCachedImage.errorBuilder}
+  final ErrorWidgetBuilder? errorBuilder;
+
+  @override
+  Widget build(final BuildContext context) {
+    final account = this.account ?? NeonProvider.of<AccountsBloc>(context).activeAccount.value!;
```

Comment:
I like this approach but you criticized it and wanted separate constructors for the default or custom account.

Reply:
Ah yeah, I forgot this :sweat_smile: